adapters/socketio_berkeley.c

```diff
@@ -650,7 +652,11 @@ int socketio_setoption(CONCRETE_IO_HANDLE socket_io, const char* optionName, con
         }
         else if (strcmp(optionName, "tcp_keepalive_time") == 0)
         {
+#ifdef __APPLE__
```

Comment:
Also one more point is that I think we should simply not support any KA options for these cases, right? Why would we allow setting only some of them?

---
In reply to: [75419327](https://github.com/Azure/azure-c-shared-utility/pull/21#discussion_r75419327) [](ancestors = 75419327,75419320)